### PR TITLE
Fixed spelling mistake in JFXDrawer

### PR DESCRIPTION
**Pull request by Sashie**
isHiding() instead of isHidding() (deprecated old method) for anyone still using it

- extremely minor but it was bugging me
- not sure why the pull request showed me the whole file not just the new lines added(first ever pr)

### CODE REVIEW

File: jfoenix/src/main/java/com/jfoenix/controls/JFXDrawer.java [hunk 1]
```diff
@@ -1,923 +1,929 @@
...
+                }
+
+                double eventPoint = 0;
+                if (directionProperty.get() == DrawerDirection.RIGHT || directionProperty.get() == DrawerDirection.LEFT) {
+                    eventPoint = e.getX();
+                } else {
+                    eventPoint = e.getY();
+                }
+
+                if (size + directionProperty.get()
```
Comment (duchuyctlk) [inline jfoenix/src/main/java/com/jfoenix/controls/JFXDrawer.java]: IMO the condition could be more readable if we write it as
```java
if (size + directionProperty.get().doubleValue() * eventPoint < activeOffset
        && (content.getCursor() == Cursor.DEFAULT || content.getCursor() == null)
        && valid == 0) {
}
```

Comment (skmedix) [inline jfoenix/src/main/java/com/jfoenix/controls/JFXDrawer.java]: [This change was made here.](https://github.com/jfoenixadmin/JFoenix/commit/9f53564b49e1db075593ab6029ee058a28bd9203#diff-ed09c98ef323d180e40af3c82a5526c9R175) But is doesn't matter if it's not breaking functionality.

**Comment by jfoenixadmin** [comment]
Hello, thanks for the pull request.
however, there is no need to keep the deprecated method. It's better to remove it.

**Comment by jfoenixadmin** [comment]
Will add it manually and remove the old method, also reformatted the file so the code is more readable.

Thanks for pointing out the issue 👍 
Regards,

**Comment by thetric** [comment]
@jfoenixadmin According to [SemVer](http://semver.org/) you should increase the next version to 2.0.0 as the removal is a breaking change:

> Given a version number MAJOR.MINOR.PATCH, increment the:
> 
> **1. MAJOR version when you make incompatible API changes,**
> 2. MINOR version when you add functionality in a backwards-compatible manner, and
> 3. PATCH version when you make backwards-compatible bug fixes.

**Pull request closed without merging**